scripts/end-to-end-test.js

```diff
@@ -0,0 +1,49 @@
+/** The purpose of this file is to do sanity checks of things that are
+ * hard to automate with unit tests. These tests don't have to (and shouldn't!)
+ * be particularly specific.
+ */
+
+const fs = require("fs");
+const path = require("path");
+const cheerio = require("cheerio");
+
+(() => {
+  /** Check that the /en-US/docs/Web/HTML/Element/video page built and
+   * is sane.
+   */
+
+  // XXX Windows?
+  const directory = "client/build/en-US/docs/Web/HTML/Element/video/";
```

Comment:
@Gregoor I know this script is only really ever run on Travis, which is Linux, but I still want it to be 100% Windows compat. Because, after all, you *can* run this script on your laptop. And here, isn't the use of `/` going to mess things up on Windows? Is the simple answer to...
```suggestion
  const directory = path.join("client", "build", "en-US", "docs", "Web", "HTML", "Element", "video");
```

Reply:
Actually, https://stackoverflow.com/a/9799490/205832 seems to say it's fine to use forward slashes since this string isn't going to be used to send to a Window sub-system. I.e. it'd be different if it was:
```javascript
const spawn = require("child_process").spawnSync;
const p = "foo/bar"
spawn("prog",  [p]);
```